File: src/main/java/user11681/limitless/config/enchantment/entry/radius/Radius.java
package user11681.limitless.config.enchantment.entry.radius;

import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject;

public class Radius {
    @CollapsibleObject(startExpanded = true)
    public HorizontalRadius xz = new HorizontalRadius();

    @CollapsibleObject(startExpanded = true)
    public VerticalRadius y = new VerticalRadius();
}

File: src/net/auoeke/limitless/config/enchantment/entry/radius/Radius.kt
package net.auoeke.limitless.config.enchantment.entry.radius

import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.CollapsibleObject

class Radius {
    @CollapsibleObject(startExpanded = true)
    var xz = HorizontalRadius()

    @CollapsibleObject(startExpanded = true)
    var y = VerticalRadius()
}
